ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSpecs.java

```diff
@@ -148,9 +151,23 @@ public static ProtocolSpecBuilder frontierDefinition(
         .blockImporterBuilder(MainnetBlockImporter::new)
         .blockHeaderFunctions(new MainnetBlockHeaderFunctions())
         .miningBeneficiaryCalculator(BlockHeader::getCoinbase)
+        .powHasher(powHasher(powAlgorithm))
         .name("Frontier");
   }
 
+  public static PoWHasher powHasher(final PowAlgorithm powAlgorithm) {
```

Comment:
This would be removed if my prior suggestion is implemented.

Reply:
removed